src/store/actions.js

```diff
@@ -192,12 +193,18 @@ export default {
 
     commit('setTipping', [contractInstanceV1, contractInstanceV2]);
   },
+
+  /**
+   * @param {{text: String}} options
+   */
   async share(_, options) {
-    await (process.env.IS_IONIC
-      ? new Promise((resolve) => window.plugins.socialsharing.shareW3C(
-        options,
-        ({ app }) => app && resolve(),
-      ))
-      : navigator.share(options));
+    // The Share API works on iOS, Android, and the Web
+    const canShare = (await Share.canShare()).value;
```

Comment:
Are there any fallback options in case we can't share?

Reply:
I didn't implement any fallback options as there weren't any originally. 
This implementation should work everywhere where the previous implementation worked, so if there was no need for a fallback option then, then I believe there is no need one now. 
If you have any ideas we could add them though.